Kwenta/kwenta
# Update the max leverage button to 25

## Description
Update the max leverage button to 25x

## Related issue


## Motivation and Context
Improve the UX

## How Has This Been Tested?
1. Select ETH market on optimism kovan and check the max leverage button is 25x
2. Select XAU market on optimism kovan and check the max leverage button is 10x

## Screenshots (if appropriate):
![image](https://user-images.githubusercontent.com/4819006/179216544-d02882fd-f1d1-4f70-a77d-31ac6ae2a4f7.png)
![image](https://user-images.githubusercontent.com/4819006/179216577-73b83ed7-0cb0-49ca-9187-03881264353e.png)
